lms/djangoapps/certificates/models.py

```diff
@@ -911,34 +941,33 @@ def set_self_generatation_enabled_for_course(cls, course_key, is_enabled):
         )
 
     @classmethod
-    def is_language_specific_templates_enabled_for_course(cls, course_key):
-        """Check whether language-specific certificates are enabled for a course.
+    def set_language_specific_templates_enabled_for_course(cls, course_key, is_enabled):
+        """Enable or disable language-specific certificates for a course.
 
         Arguments:
             course_key (CourseKey): The identifier for the course.
-
-        Returns:
-            boolean
+            is_enabled (boolean): Whether to enable or disable language-specific certificates.
 
         """
-        try:
-            latest = cls.objects.filter(course_key=course_key).latest()
-        except cls.DoesNotExist:
-            return False
-        else:
-            return latest.language_specific_templates_enabled
+        default = {
+            'language_specific_templates_enabled': is_enabled,
+        }
+        CertificateGenerationCourseSetting.objects.update_or_create(
+            course_key=course_key,
+            defaults=default
+        )
 
     @classmethod
-    def set_language_specific_templates_enabled_for_course(cls, course_key, is_enabled):
-        """Enable or disable language-specific certificates for a course.
+    def set_include_hours_of_effort_for_course(cls, course_key, is_included):
```

Comment:
Where are these setters called from?

Reply:
Just in the unit tests. Here is an example
https://github.com/edx/edx-platform/blob/McKenzieW/course-cert-hours-of-effort/lms/djangoapps/certificates/tests/test_webview_views.py#L1335
all of the calls to the 'set' methods are in that file